mtags/src/main/scala-2/scala/meta/internal/pc/completions/ScaladocCompletions.scala

```diff
@@ -208,7 +208,8 @@ trait ScaladocCompletions { this: MetalsGlobal =>
           true
       }
       if (isEligible) {
-        if (t.pos.isDefined && t.pos.start >= pos.start) associatedDef = Some(t)
+        if (t.pos.isDefined && t.pos.focusStart.line == pos.line + 1)
```

Comment:
Not sure if I understand the code correctly, but would it be a good idea to skip empty lines here, so that they don't count? I don't know if it would make sense, maybe it's not intened on purpose, wanted to ask just in case. :slightly_smiling_face:

Reply:
It's a good question. We already show this completion only in the comment is one line above the definition in Scala 3, so I wanted to make it consistent. I think in cases like 

```scala

/**@@             <-- we probably don't want docstring about `method` here



def method(a: Int) = ???
```

Reply:
Thanks, makes sense, plus I didn't know about the Scala 3 difference. :+1: